### PR TITLE
classicube: init at 1.3.2

### PR DESCRIPTION
**Pull request by 360ied**
###### Description of changes

ClassiCube is a lightweight, custom Minecraft Classic/Classicube client with optional additions written from scratch in C.

Homepage: https://www.classicube.net/

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by 360ied** [comment]
I am the author of all the patches.

**Comment by 360ied** [comment]
Since this hasn't received any attention in the past 5 days,
I'm pinging all the participants in the last PR for ClassiCube.
(https://github.com/NixOS/nixpkgs/pull/67569)

@marius851000 @mmahut @kira-bruneau @SuperSandro2000 @lheckemann @disassembler

File: pkgs/games/classicube/default.nix [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+, fetchFromGitHub
+  # native build dependencies
+, dos2unix
+, makeWrapper
+  # build dependencies
+, SDL2
+, libGL
+, curl
+, openal
+  # runtime dependencies
```
Comment (SuperSandro2000) [inline pkgs/games/classicube/default.nix:12]:
```suggestion
, dos2unix
, makeWrapper
, SDL2
, libGL
, curl
, openal
```
thats already documented below and the comments can become out of date

File: pkgs/games/classicube/default.nix [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      --replace 'JOBS=1' "JOBS=$NIX_BUILD_CORES"
+  '';
+
+  buildInputs = [ SDL2 libGL curl openal liberation_ttf ];
+
+  buildPhase = ''
+    cd 'src'
+    make
+    cd -
+  '';
```
Comment (SuperSandro2000) [inline pkgs/games/classicube/default.nix:70]: Please change the directory in preBuild.

File: pkgs/games/classicube/default.nix [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+  enableParallelBuilding = true;
+
+  postPatch = ''
+    # ClassiCube hardcodes locations of fonts.
+    # This changes the hardcoded location
+    # to the path of liberation_ttf instead
+    substituteInPlace src/Platform_Posix.c \
+      --replace '%NIXPKGS_FONT_PATH%' "${font_path}"
+  '' + lib.optionalString enableParallelBuilding ''
```
Comment (SuperSandro2000) [inline pkgs/games/classicube/default.nix:57]:
```suggestion

```

File: pkgs/games/classicube/default.nix (hunk 2 above)
Comment (360ied) [reply pkgs/games/classicube/default.nix]: Done!

**Pull request merged**